src/Helpers.js

```diff
@@ -35,3 +35,25 @@ export const getControlData = (component = {}, controlId) => {
 
   return controlData;
 };
+
+export const formatCatalogTitle = (catalogTitle) => {
+  let titleArray = catalogTitle.split("_");
+  if (titleArray.length == 5) {
+    let formattedImpactLevel =
+      titleArray[4][0].toUpperCase() + titleArray[4].substring(1).toLowerCase();
+    return (
+      titleArray[0] +
+      " " +
+      titleArray[1] +
+      " " +
+      titleArray[2] +
+      "." +
+      titleArray[3] +
+      " (" +
+      formattedImpactLevel +
+      ")"
+    );
+  } else {
+    return catalogTitle;
```

Comment:
Should an error be raised if the catalog title can't be parsed?

Reply:
I wasn't thinking of raising an error because this is only display text and we wouldn't do anything with the error information.  However, I could add `if(titleArray.length ==5 && (typeOf titleArray) == String){}` to really lock it down and everything else would fall through to be sure that there would be no errors.  Does that sound satisfying?

Reply:
Hmm, if we want to start handling that kind of functionality and being really careful about typing variables, then it'd probably be worth talking about using typescript.

In this particular case, I don't think we need to raise an error, unless we have some very specific thing we plan on doing with it. Since the function itself is really low risk and is more a cosmetic benefit than anything functional, I think it's perfectly fine here to pass along the original catalog name.

Reply:
@amyd11 makes some good points, so disregard!